src/psyclone/tests/psyir/frontend/fparser2_program_handler_test.py

```diff
@@ -42,48 +42,39 @@
 
 from fparser.common.readfortran import FortranStringReader
 from psyclone.errors import GenerationError
-from psyclone.psyir.nodes import Container
+from psyclone.psyir.nodes import FileContainer
```

Comment:
Unused pytest and GenerationError imports

Reply:
Oh dear. Note to self. Check all files before sending to review.